### DISCUSSION TITLE
IBEX on human bone marrow sample

### DISCUSSION THREAD

**Discussion by NT0501** [IBEXImagingCommunity/ibex_imaging_knowledge_base#143] [Q&A] [answered]
Hello everyone,

I would like to apply IBEX on human bone marrow samples in my experimental design.
Most of them are FFPE samples and some contain fixed frozen samples.
Have you ever performed IBEX on human bone marrow samples? 
If there are any precautions, I would appreciate it if you could let me know.

Thanks a lot.

**Comment by radtkea** [comment]
Hello!

Thank you for your question and interest in applying IBEX to human FFPE bone marrow samples.

I do not have much experience with imaging human bone marrow samples. Here's all I know:)
1. Fixed frozen bone marrow biopsies: They floated right off the slide when I rehydrated them and after multiple washes. 
2. FFPE: This is a better way to preserve the tissue architecture but BM FFPE samples are HIGHLY autofluorescent!

Members of the lab have adopted IBEX to fixed frozen BM samples from mice. It required significant optimization because of tissue-specific challenges.

IBEX imaging of human FFPE samples is definitely possible, but I think you will need to:
1. Work to aggressively reduce autofluorescence (AF). Colleagues eliminate AF using a photo-irradiation protocol prior to IBEX of lung FFPE samples. Here is a [paper](https://meridian.allenpress.com/aplm/article/143/3/362/9257/Elimination-of-Autofluorescence-in-Archival) summarizing different strategies to reduce AF in human FFE bone marrow samples. Here is another [paper](https://journals.sagepub.com/doi/10.1177/002215540405200305?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed) detailing decalcification and AF correction.
2. In general, I have found attempts to reduce AF (SudanBlack, TrueView, borohydride) to be imperfect for a great number of reasons. I prefer to eliminate it in the beginning (photo-irradiation), amplify signal, and deal with any residual AF computationally (background subtraction) using [channel arithmetics](https://niaid.github.io/imaris_extensions/XTChannelArithmetic.html). 

Strategies to amplify signal:
1. [Opal-plex](https://www.pnas.org/doi/full/10.1073/pnas.2018488117): Dr. Emily Speranza incorporated the TSA-based opal system with IBEX to image heavily fixed tissues. This could be an option for you. Here is a [paper](https://www.sciencedirect.com/science/article/pii/S0076687919302368#s0015) that does single cycle opal imaging in the bone marrow.
2. [Cell IDx](https://www.leicabiosystems.com/us/life-sciences-and-research-solutions/cell-idx/) for selected targets. 
3. I haven't tried this approach from [Cell Signaling Technology](https://www.cellsignal.com/applications/signalstar-multiplex-ihc-overview) but it looks promising. 

Happy to connect with you about your specific panel and other issues. Just email me.

**Reply by NT0501** [reply, marked as answer]
I really appreciate your help.
I will try it!
